### PR TITLE
feat(matomo-analytics): export apifactory for dynamic plugin

### PR DESCRIPTION
**Pull request by deshmukhmayur**
Adding dynamic plugin support for the matomo analytics module.

### CODE REVIEW

**Review by gashcrumb — APPROVED** [review]

File: plugins/analytics-module-matomo/src/index.ts [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+  configApiRef,
+  createApiFactory,
+} from '@backstage/core-plugin-api';
+
+import { MatomoAnalytics } from './api';
+
 export { analyticsModuleMatomoPlugin } from './plugin';
 export * from './api';
+
+export const MatomoAnalyticsApi = createApiFactory({
```
Comment (gashcrumb) [inline plugins/analytics-module-matomo/src/index.ts:12]: LGTM though I'd suggest to keep this instantiation either alongside your plugin declaration in ./plugin.ts or in a file within ./api and just re-export the modules from this file.

Comment (deshmukhmayur) [reply plugins/analytics-module-matomo/src/index.ts]: Makes sense. Updated the commit.

**Review by gashcrumb — APPROVED** [review]
:+1:

**Comment by invincibleJai** [comment]
@deshmukhmayur thanks for the PR, can you pls add `OWNERS` file to mataomo plugin as well as you have for https://github.com/janus-idp/backstage-plugins/blob/main/plugins/feedback/OWNERS 

This will let your team approve the PR's going forward

**Comment by deshmukhmayur** [comment]
> @deshmukhmayur thanks for the PR, can you pls add `OWNERS` file to mataomo plugin as well as you have for https://github.com/janus-idp/backstage-plugins/blob/main/plugins/feedback/OWNERS
> 
> This will let your team approve the PR's going forward

Ack. @invincibleJai I have added the OWNERS file for the plugin.

**Pull request merged**